Production/src/BaseTupleProducer.cc

```diff
@@ -681,7 +697,30 @@ void BaseTupleProducer::SelectVetoElectron(const ElectronCandidate& electron, Cu
     cut(electron_dxy < dxy, "dxy", electron_dxy);
     const double electron_dz = std::abs(electron->gsfTrack()->dz(primaryVertex->position()));
     cut(electron_dz < dz, "dz", electron_dz);
-    const float passID = electron->electronID("mvaEleID-Fall17-iso-V1-wpLoose");
+    const float passID = electron->electronID("mvaEleID-Fall17-iso-V2-wpLoose");
+    cut(passID > 0.5, "electronId");
+    for(size_t n = 0; n < signalElectrons.size(); ++n) {
+        std::ostringstream ss_name;
+        ss_name << "isNotSignal_" << n + 1;
+        const bool isNotSignal =  &(*electron) != &(*(*signalElectrons.at(n)));
+        cut(isNotSignal, ss_name.str());
+    }
+}
+
+void BaseTupleProducer::SelectTightVetoElectron(const ElectronCandidate& electron, Cutter& cut,
```

Comment:
use the same method for the tight and the baseline vetos to avoid duplications and possible discrepancies in the future.